src/array/methods.jl

```diff
@@ -131,15 +131,15 @@ end
     """
     function Base.eachslice(A::AbstractDimArray; dims)
         dimtuple = _astuple(dims)
-        if !(dimtuple == ()) 
+        if !(dimtuple == ())
```

Comment:
Would be nice to remove all these whitespace changes so we can see the real changes